app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainViewModel.kt

```diff
@@ -13,7 +13,7 @@ class MainViewModel : ViewModel() {
     var stock: MutableLiveData<ArrayList<Stock>> = MutableLiveData<ArrayList<Stock>>()
     var stockService: StockService = StockService()
     private lateinit var firestore : FirebaseFirestore
-    private var storageReferenence = FirebaseStorage.getInstance().getReference()
+    private var storageReference = FirebaseStorage.getInstance().getReference()
```

Comment:
This reference is unused but because someone already said that, I'll just point out that it was misspelled.